Description: putting the paths of gluegen and ant-contrib jars in the ant
 build files
Author: Sylvestre Ledru <sylvestre@debian.org>
Forwarded: not-needed
Last-Update: 2023-04-17

--- a/make/build-common.xml
+++ b/make/build-common.xml
@@ -19,7 +19,7 @@
     </fail>
 
     <!-- Pull in GlueGen cpptasks build file -->
-    <property name="gluegen.root" value="../../gluegen" />
+    <property name="gluegen.root" value="/usr/share/gluegen2/" />
     <import file="${gluegen.root}/make/gluegen-cpptasks.xml" />
     <import file="${gluegen.root}/make/jogamp-archivetasks.xml" />
     <import file="${gluegen.root}/make/jogamp-androidtasks.xml" />
@@ -28,7 +28,7 @@
     <!-- Add JOAL (optional) -->
     <property name="joal.root" value="../../joal" />
 
-    <property name="ant-contrib.jar"  value="${gluegen.root}/make/lib/ant-contrib-1.0b3.jar" />
+    <property name="ant-contrib.jar"  value="/usr/share/java/ant-contrib.jar" />
 
     <condition property="rootrel.build" value="build">
       <not>
@@ -237,7 +237,7 @@
         <property name="gluegen.build.xml"          value="${gluegen.make}/build.xml" />
         <property name="build.gluegen"              value="${gluegen.root}/${rootrel.build}" />
         <property name="obj.gluegen"                value="${build.gluegen}/obj"/>
-        <property name="gluegen.jar"                value="${build.gluegen}/gluegen.jar" />
+        <property name="gluegen.jar"                value="/usr/share/java/gluegen2.jar" />
 
         <property name="build.joal"                 value="${joal.root}/${rootrel.build}" /> <!-- optional -->
 
@@ -507,7 +507,7 @@
 
 
         <!-- JavaSE combinations -->
-        <property name="gluegen-rt.jar"                 value="${build.gluegen}/gluegen-rt.jar" />
+        <property name="gluegen-rt.jar"                 value="/usr/share/java/gluegen2-rt.jar" />
         <property name="gluegen-rt-android.jar"         value="${build.gluegen}/gluegen-rt-android.jar" />
         <property name="gluegen-test-util.jar"          value="${build.gluegen}/gluegen-test-util.jar" />
         <property name="jogl-test.jar"                  value="${jar}/jogl-test.jar"/>
@@ -828,7 +828,10 @@
 
         <!-- Run the GlueGen build to ensure that the GlueGen ANT task
            - has been built. -->
-        <ant antfile="${gluegen.build.xml}" dir="${gluegen.make}" target="base.compile" inheritAll="false"/>
+        <ant antfile="${gluegen.build.xml}" dir="${gluegen.make}" target="base.compile" inheritAll="false">
+          <property name="build" value="${basedir}/../"/>
+          <property name="test.dir" value="${basedir}/../test"/>
+        </ant>
     </target>
 
 
--- a/make/build.xml
+++ b/make/build.xml
@@ -355,7 +355,7 @@
     </target>
 
     <target name="tag.build" depends="init">
-        <copy file="${build.gluegen}/artifact.properties" todir="${build}" overwrite="true"/>
+        <copy file="/usr/share/gluegen2/artifact.properties" todir="${build}" overwrite="true"/>
         <echo message='jogl.build.version=${jogamp.version}${line.separator}'   file="${build}/artifact.properties" append="true"/>
         <echo message='jogl.build.number=${jogl.build.number}${line.separator}' file="${build}/artifact.properties" append="true"/>
         <echo message='jogl.build.id=${jogl.build.id}${line.separator}'         file="${build}/artifact.properties" append="true"/>
